### ISSUE TITLE
Fritzcap stoped working after firmware upgrade to Fritz!OS 7.50

### ISSUE THREAD

**Issue by chunter1** [jpluimers/fritzcap#39]
I just updated my FritzBox to firmware version 7.50 and fritzcap does not work anymore.
The problem is, that i now get .cap files with size 0.
Is there any solution?

**Comment by JackSlaterIV** [comment]
Yes, I confirm this issue. 
I have found a workaround in this post: https://www.ip-phone-forum.de/threads/fritzcap-tool-f%C3%BCr-etherreal-trace-und-audiodaten-extraktion-v2-0.232682/post-2512522

**Comment by gtrooper** [comment]
@JackSlaterIV Did this workaround actually works ? Can you elaborate ? I could not make it work ! I run FW 7.56 on a FritzBox 7590 AX v2 ! TY in advance !

**Comment by JackSlaterIV** [comment]
> 

Yes, It works for me. I could share my implementation but it's built on my python 3 branch (see my pull request here).

**Comment by gtrooper** [comment]
The issue I have is not with Python 3. It works amazingly well in Python 2 although the change to 3 is the right direction for the future. The issue I am facing is AFTER FW Version 7.21 (e.g. 7.29 and later) I face the issue which the recording starts but after the call is done there is NO WAV file… I applied all the changes they mention in the ip phone site you linked but still it does not work. I Disabled HD Telephony on all devices and still the WAV file does not get created. What did you do to solve this crucial part ? TY so much for your contribution !

**Comment by JackSlaterIV** [comment]
> The issue I have is not with Python 3. It works amazingly well in Python 2 although the change to 3 is the right direction for the future. The issue I am facing is AFTER FW Version 7.21 (e.g. 7.29 and later) I face the issue which the recording starts but after the call is done there is NO WAV file… I applied all the changes they mention in the ip phone site you linked but still it does not work. I Disabled HD Telephony on all devices and still the WAV file does not get created. What did you do to solve this crucial part ? TY so much for your contribution !

Hello, as already said I am using a different branch (after my python 3 conversion), so I am guessing how it could work in your case.

You can try to remove "+self.cap_interface" in the line 264 of capture_monitor.py.

In fritzcap.conf use these lines:
start_str       = ?capture=Start&snaplen=1600&filter=
stop_str        = ?iface=internet&minor=17&type=3&capture=Stop

When fritzcap is called, the interface 3-17 should be specified:
fritzcap.py --capture_files --decode_files --monitor_calls --cap_interface 3-17 --box_name 192.xxx.xxx.1 --user xxxxxxx --password xxxxxxxx

Try and let me know.

**Comment by gtrooper** [comment]
Hi and thank you so much for your answer ! Sadly it did not work ! I had already changed what you proposed me to change but I tried it again in case I had messed up something and it did not work. The log says INVALID PCAP HEADER. The last version it works with zero issues for me was FritzOS 7.21 ! Ty again !

**Comment by kuzco1** [comment]
Do we have any news on that ?
Running 7.50 on a Fritz7590 an the above command:
fritzcap.py --capture_files --decode_files --monitor_calls --cap_interface 3-17 --box_name 192.xxx.xxx.1 --user xxxxxxx --password xxxxxxxx
Recording working fine, but wireshark does not found any RTSP call.
